plugins/MonitorStage/MonitorStage.py

```diff
@@ -92,46 +72,3 @@ def _updateSidebar(self):
         # TODO: currently the sidebar component for prepare and monitor stages is the same, this will change with the printer output device refactor!
         sidebar_component_path = os.path.join(Resources.getPath(Application.getInstance().ResourceTypes.QmlFiles), "Sidebar.qml")
         self.addDisplayComponent("sidebar", sidebar_component_path)
-
-    def _updateIconSource(self):
-        if Application.getInstance().getTheme() is not None:
-            icon_name = self._getActiveOutputDeviceStatusIcon()
-            self.setIconSource(Application.getInstance().getTheme().getIcon(icon_name))
-
-    ##  Find the correct status icon depending on the active output device state
-    def _getActiveOutputDeviceStatusIcon(self):
-        # We assume that you are monitoring the device with the highest priority.
-        try:
-            output_device = Application.getInstance().getMachineManager().printerOutputDevices[0]
-        except IndexError:
-            return "tab_status_unknown"
-
-        if not output_device.acceptsCommands:
-            return "tab_status_unknown"
-
-        if output_device.activePrinter is None:
-            return "tab_status_connected"
-
-        # TODO: refactor to use enum instead of hardcoded strings?
-        if output_device.activePrinter.state == "maintenance":
-            return "tab_status_busy"
-
-        if output_device.activePrinter.activePrintJob is None:
-            return "tab_status_connected"
-
-        if output_device.activePrinter.activePrintJob.state in ["printing", "pre_print", "pausing", "resuming"]:
-            return "tab_status_busy"
-
-        if output_device.activePrinter.activePrintJob.state == "wait_cleanup":
-            return "tab_status_finished"
-
-        if output_device.activePrinter.activePrintJob.state in ["ready", ""]:
-            return "tab_status_connected"
-
-        if output_device.activePrinter.activePrintJob.state == "paused":
-            return "tab_status_paused"
-
-        if output_device.activePrinter.activePrintJob.state == "error":
-            return "tab_status_stopped"
-
-        return "tab_status_unknown"

```

Comment:
I made these same changes this morning with Luke but wasn't fast enough to push them before you did. Question though: Do we want to delete it or just comment it out? This is functionality we may want reuse in the near future related to sync.

I mean I guess we could copy it from earlier commits but that can be kind of a pain.

Reply:
The decision was to remove it. We can always revert this commit and go back to the previous code.